libs/net/dal/Migrations/1.0.8/Up/PreUp/01-tag.sql

```diff
@@ -0,0 +1,38 @@
+DO $$
+BEGIN
+
+-- Remove foreign key to allow editing the tag primary key.
+ALTER TABLE public."content_tag" DROP CONSTRAINT "FK_content_tag_tag_tag_id";
```

Comment:
All this was required to fix the `tag` table bug.  We couldn't add new tags because the primary key was not a identity seed.